src/wurf/git_semver_resolver.py

```diff
@@ -47,28 +52,30 @@ def resolve(self):
                     "were {}".format(self.dependency.major, tags),
                 dependency=self.dependency)
 
-        # Use the parent folder of the path retuned to store different
-        # versions of this repository
-        repo_folder = os.path.dirname(path)
-        tag_path = os.path.join(repo_folder, tag)
+        # Use the path retuned to create a unique location for this checkout
+        repo_hash = hashlib.sha1(path.encode('utf-8')).hexdigest()[:6]
```

Comment:
@mortenvp The same thing here. I don't want to see any extra hash code after the tag.